### PR TITLE
updates delete confirmation dialog

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/7837
## Testing
**URL to test:** 
https://deploy-preview-493--atd-moped-main.netlify.app/
**Steps to test:**
go to projects page, select a project, select the ellipsis in the upper right corner, click delete

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [ ] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)~ n/a

### CODE REVIEW

File: moped-editor/src/components/GridTable/GridTable.js [hunk 1]
```diff
@@ -401,6 +401,8 @@ const GridTable = ({ title, query, searchTerm }) => {
     query.config.options.useQuery
   );
 
+  console.log(data);
```
Comment (mateoclarke) [inline moped-editor/src/components/GridTable/GridTable.js:404]: Let's remove these console logs, plz

File: moped-editor/src/views/projects/projectView/ProjectStatusBadge.js [hunk 2]
```diff
@@ -164,6 +164,8 @@ const useChipStyles = makeStyles(theme => ({
  * @constructor
  */
 const ProjectStatusBadge = ({ status, phase, projectStatuses }) => {
+  console.log(status);
+  console.log(phase);
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectStatusBadge.js:168]: same here on console logs

**Review by mateoclarke — APPROVED** [review]
Looks good. Once you clean up those added console logs, we're good to merge.
🚢💨✨ 

lizzie-mcguire-delete

**Review by amenity — APPROVED** [review]
LSTM!✨ 🙏

**Pull request merged**